bitwarden_license/src/app/organizations/manage/sso.component.ts

```diff
@@ -117,9 +117,11 @@ export class SsoComponent implements OnInit {
     }
 
     async submit() {
-        if (!this.keyConnectorIsValid || this.keyConnectorUrl.dirty)
-        {
-            await this.testKeyConnector();
+        if (this.data.get('keyConnectorEnabled').value) {
+            if (this.keyConnectorUrl.dirty || this.keyConnectorIsValid == null) {
```

Comment:
I think we need to check if `keyConnectorEnabled` is dirty as well, since someone might have a url set but disabled and then turn it on.